mapeditor/inspector/questwidget.cpp

```diff
@@ -439,3 +440,86 @@ bool QuestDelegate::eventFilter(QObject * object, QEvent * event)
 	}
 	return QStyledItemDelegate::eventFilter(object, event);
 }
+
+void QuestDelegate::updateModelData(QAbstractItemModel * model, const QModelIndex & index) const
+{
+	QStringList textList;
+	textList += QObject::tr("Quest:");
+	textList += QObject::tr("Day of Week: %1").arg(quest.mission.dayOfWeek);
+	textList += QObject::tr("Days Passed: %1").arg(quest.mission.daysPassed);
+	textList += QObject::tr("Hero Level: %1").arg(quest.mission.heroLevel);
+	textList += QObject::tr("Hero Experience: %1").arg(quest.mission.heroExperience);
+	textList += QObject::tr("Mana Points: %1").arg(quest.mission.manaPoints);
+	textList += QObject::tr("Mana Percentage: %1").arg(quest.mission.manaPercentage);
+	textList += QObject::tr("Primary Skills: %1/%2/%3/%4").arg(quest.mission.primary[0]).arg(quest.mission.primary[1]).arg(quest.mission.primary[2]).arg(quest.mission.primary[3]);
+
+	QStringList resourcesList;
+	for(GameResID resource = GameResID::WOOD; resource < GameResID::COUNT ; resource++)
+	{
+		if(resource == GameResID::MITHRIL)
+			continue;
+		if(quest.mission.resources[resource] == 0)
+			continue;
+		MetaString str;
+		str.appendName(resource);
+		resourcesList += QString::fromStdString(str.toString()) + ": " + QString::number(quest.mission.resources[resource]);
+	}
+	textList += QObject::tr("Resources: ") + resourcesList.join(", ");
```

Comment:
is it ok to have a single long string? Also better to make it a param of the user-visible string. (applies to all other similar places)

Reply:
I think that in most cases those lists will have one or two entities, and then it's easier to see that some name is related to artifact or creature. Example how it look if all resources are required
![image](https://github.com/user-attachments/assets/a5a7f610-792e-4454-814a-540f303045e3)

Reply:
OK, let's leave like that now, can always be adjusted later if necessary